frontend/src/pages/interviewschedule.js

```diff
@@ -119,16 +141,48 @@ class InterviewSchedule extends Component {
         <hr />
         <Container>
           <form onSubmit={this.uploadSchedule}>
-            <>
+            <div>
```

Comment:
Prefer `<>` over `<div>`

Reply:
i would mention that it's `nit` for these, especially when you would preface it with "Prefer"